src/api/urls.js

```diff
@@ -1,6 +1,6 @@
 // Defining URLS object for manipulation...
 import { IS_PROD, IS_CANARY, IS_LOCAL } from "../config";
-
+export const MASSENERGIZE_PRODUCTION_URL = "https://community.massenergize.org";
```

Comment:
Would it be better to use URLS["community"] with different versions for Dev, Canary and production?  I think ok for now but it will confuse testing of the feature in canary a bit comfusing.